ChemistryLib/PhreeqcKernelData/EquilibriumReactants.h

```diff
@@ -0,0 +1,60 @@
+/**
+ * \file
+ * \copyright
+ * Copyright (c) 2012-2020, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#pragma once
+
+#include <set>
+#include <string>
+#include <vector>
+
+#include <iphreeqc/src/src/phreeqcpp/PPassemblage.h>
+
+namespace ChemistryLib
+{
+namespace PhreeqcKernelData
+{
+class PhaseComponent final : private cxxPPassemblageComp
+{
+public:
+    PhaseComponent(std::string&& name_, double const initial_amount,
+                   double const saturation_index);
+
+    cxxPPassemblageComp const* castToBaseClass() const
+    {
+        return static_cast<cxxPPassemblageComp const*>(this);
+    }
+
+    std::string const getName() const { return Get_name(); }
```

Comment:
The phreeqc's function is returning a copy or reference?

Reply:
Good catch. Return a reference. Fixed.